dotcom-rendering/src/web/layouts/CommentLayout.tsx

```diff
@@ -322,6 +322,7 @@ export const CommentLayout = ({
 						showTopBorder={false}
 						showSideBorders={false}
 						padded={false}
+						shouldCenter={false}
```

Comment:
```suggestion
```

Just a thought - since `shouldCenter` is an optional prop could we delete this here? It might avoid any confusion about the fact the ad ends up centred in the slot.

Reply:
Ignore that it defaults to true doesn't it!